.travis.yml

```diff
@@ -101,7 +101,7 @@ matrix:
     - otp_release: 20.0
       env: PRESET=pgsql_mnesia DB=pgsql REL_CONFIG="with-pgsql with-jingle-sip"
            RUN_SMALL_TESTS=true
-    - otp_release: 18.3
+    - otp_release: 21.0
```

Comment:
Maybe we should follow the existing pattern, where the most recent OTP version is applied to Jingle/SIP job and Dialyzer? Experience shows that the former is risky with new OTPs... :)

Reply:
Yes, I totally agree. I'll add Jingle/SIP tests. Regarding Dialyzer, I think it'd be better to do in a separate PR. I tried it and observed 20 or so errors...